lib/motion-markdown-it-plugins/emoji/plugin/data/light.rb

```diff
@@ -0,0 +1,161 @@
+module MotionMarkdownItPlugins
+  module EmojiPlugin
+    module Data
+      module Light
+        EMOJIES_DEF_LIGHT = {
+          "grinning" => "😀",
```

Comment:
What do you think about using ruby symbols as hash keys? If you want avoid hashrocket syntax (`=>`)

Reply:
Because of these in `full.rb`

```ruby
          "100" => "💯",
          "1234" => "🔢",
```

we can't convert all of them into symbols.  However I did make a change to allow the options passed in to be symbols.